Makefile

```diff
@@ -46,9 +46,9 @@ install:
 	touch $(DESTDIR)/usr/lib/clock-epoch
 
 	# only generate manifest and dpkg.yaml files for lp build
-	if [ -e /build/core22 ]; then \
-		/bin/cp $(DESTDIR)/usr/share/snappy/dpkg.list /build/core22/core22-$$(date +%Y%m%d%H%M)_$(DPKG_ARCH).manifest; \
-		/bin/cp $(DESTDIR)/usr/share/snappy/dpkg.yaml /build/core22/core22-$$(date +%Y%m%d%H%M)_$(DPKG_ARCH).dpkg.yaml; \
+	if [ -e /build/core24 ]; then \
```

Comment:
Could we have a makefile variable to avoid us repeting the current base too much?

Reply:
Done